Runtimes/Supplemental/Synchronization/cmake/caches/Vendors/Apple/arm64-MacOSX.cmake

```
@@ -0,0 +1,18 @@
+if(NOT DEFINED CMAKE_OSX_DEPLOYMENT_TARGET)
+  message(SEND_ERROR "CMAKE_OSX_DEPLOYMENT_TARGET not defined")
+endif()
+
+if(NOT DEFINED SwiftSynchronization_TARGET_VARIANT_DEPLOYMENT_TARGET)
+  message(WARNING "SwiftSynchronization_TARGET_VARIANT_DEPLOYMENT_TARGET not defined")
```

We may want to make this an error instead of just a warning.

```suggestion
  message(SEND_ERROR "SwiftSynchronization_TARGET_VARIANT_DEPLOYMENT_TARGET not defined")
```